app/src/main/java/org/simple/clinic/facility/alertchange/AlertFacilityChangeSheet.kt

```diff
@@ -0,0 +1,49 @@
+package org.simple.clinic.facility.alertchange
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import kotlinx.android.synthetic.main.sheet_alert_facility_change.*
+import org.simple.clinic.R
+import org.simple.clinic.facility.Facility
+import org.simple.clinic.widgets.BottomSheetActivity
+
+class AlertFacilityChangeSheet : BottomSheetActivity() {
+
+  companion object {
+    private const val CURRENT_FACILITY = "current_facility"
+
+    fun intent(context: Context, currentFacility: Facility): Intent {
+      val intent = Intent(context, AlertFacilityChangeSheet::class.java)
+      intent.putExtra(CURRENT_FACILITY, currentFacility)
+      return intent
+    }
+  }
+
+  private val currentFacility by lazy {
```

Comment:
Maybe we should pass `Facility.name` instead of the entire `Facility` object? wdyt?

Reply:
Yes. I will do this.